docs/tutorial/force_calibration.rst

```diff
@@ -104,3 +118,86 @@ We can plot the calibration by calling::
+
+.. math::
+
+    \gamma(f, R/l) = \frac{\gamma_{stokes}(f)}{1 - \frac{9}{16}\frac{R}{l}\left(1 - \left((1 - i)/3\right)\sqrt{\frac{f}{f_{\nu}}} + \frac{2}{9}\frac{f}{f_{\nu}}i - \frac{4}{3}(1 - e^{-(1-i)(2l-R)/\delta})\right)}
+
+where :math:`\delta = R \sqrt{\frac{f_{\nu}}{f}}` and :math:`R` corresponds to the bead radius, :math:`l` to the distance from the bead center to the nearest surface.
+
+While these models may look daunting, they are all available in Pylake and can be used by simply providing a few additional arguments to the :class:`~.PassiveCalibrationModel`.
+It is recommended to use these equations when less than 10% systematic error is desired :cite:`tolic2006calibration`.
+
+The figure below shows the difference between the hydrodynamically correct spectrum and the regular Lorentzian for various bead sizes.
+
+.. image:: hydro.png
+
+These more advanced models require a few extra parameters namely the density of the sample, density of the bead and distance to the surface (in meters)::
+
+    force_model = lk.PassiveCalibrationModel(bead_diameter, hydrodynamically_correct=True, rho_sample=999, rho_bead=1060.0, distance_to_surface=1e-6)
+
+Note that when `rho_sample` and `rho_bead` are omitted, values for water and polystyrene are used for the sample and bead density respectively.
+
+Additionally, when the parameter `distance_to_surface` is omitted, a simpler model is used which assumes the experiment was performed deep in bulk (neglecting the increased drag induced by the nearby surface).
+
+Fast Sensors
+------------
+
+Fast detectors provide a higher bandwidth, meaning that they do not need the effect of the diode in the model.
+
+We can omit the diode response model by passing `fast_sensor=True` to the `CalibrationModel`::
+
+    force_model = lk.PassiveCalibrationModel(bead_diameter, viscosity=0.001002, temperature=20, fast_sensor=True, hydrodynamically_correct=True)
+
+Note however, that this makes using the hydrodynamically correct model critical, as the Lorentzian doesn't actually capture the data very well.
+This is illustrated in the figure below.
+Here we see the same power spectrum (acquired on a fast detector) fitted with three different models.
+
+.. image:: fast_hydro.png
+
+Here we can see that the fit with the Lorentzian model with the diode filtering effect seems to fit the data quite well.
+However, when we enable the `fast_sensor` flag, we see that the Lorentzian model doesn't actually describe the data.
+With the diode model disabled, we obtain a very biased fit.
+Enabling the hydrodynamic corrections, we can see that this describes the power spectrum much better.
+Note that for this particular data set the Lorentzian with diode model and hydrodynamically correct model without diode model result in very similar stiffness estimates, no general statement can be made about the bias the former approach occurs in different conditions.
```

Comment:
This sentence is a bit confusing, let me see if it is correct:

It means using the passive model with `hydrodynamically_correct=True` and `fast_sensor=False` may actually give reasonable results (like the ones in the example) but we don't guarantee anything?

Reply:
Hm, no, that's not what I meant with that phrase. Let me rephrase that and see if this is better:

_If we compare the different fits, we can see that the Lorentzian model with diode effect (`fast_sensor=False`) gives similar stiffness estimates as the hydrodynamically correct model without the diode effect. While this is true for this particular dataset, no general statement can be made about the bias of fitting a Lorentzian rather than the hydrodynamically correct power spectrum. If low bias is desired, one should use the hydrodynamically correct model. On regular sensors, it is best to fit the hydrodynamically correct model with the diode model enabled._

But you raise a good additional point. In this case, fitting `hydrodynamically_correct=True` with `fast_sensor=False` actually _also_ gave a good result (it estimated `alpha` to one, effectively disabling the diode effect during the fitting procedure). Again though, I would not recommend that procedure either if you are dealing with a fast sensor.